charts/sourcegraph-executor/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: sourcegraph-executor
+description: Chart for installing Sourcegraph executors
+icon: https://sourcegraph.com/favicon.ico
+type: application
+
+# Chart version, separate from Sourcegraph
+version: "4.4.1"
```

Comment:
Yes and no, I've followed the migrator example here. Given also that in fact this will be updated alongside the release, and executor versions are tied to the same. I'd vote in favor of keeping these in line, making it easier for end users to deploy. 

But happy to be challenged on this given it's the chart version, not the version of the images that are packaged.

Reply:
Makes sense to follow whatever is the current convention!